src/js/common/schemaform/validation.js

```diff
@@ -95,9 +98,9 @@ export function uiSchemaValidate(errors, uiSchema, schema, formData, formContext
             }
           };
         }
-        uiSchemaValidate(errors, uiSchema.items, schema.items, formData, formContext, newPath);
+        uiSchemaValidate(errors, uiSchema.items, schemaWithDefinitions.items, definitions, formData, formContext, newPath);
       });
-    } else {
+    } else if (!uiSchema.items) {
       Object.keys(uiSchema)
         .filter(prop => !prop.startsWith('ui:'))
         .forEach((item) => {
```

Comment:
If you mean submitting on the review page, that's something I expect to be broken right now. Submit is being addressed in #4811.